.github/workflows/CI_dev_environment.yml

```
@@ -0,0 +1,55 @@
+name: CI Test dev environment
+
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+
+  test_dev:
+    name: test dev environment / python-${{ matrix.python-version }} / ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        python-version: ['3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - name: Create conda environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
+      - name: Show conda config
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
+          conda config --show-sources
+          conda config --show
+          conda env list
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python
+          python --version
+      - name: Show environment variables
+        shell: bash -l {0}
+        run: |
+          env | sort
+      - name: Install conda dependencies
+        shell: bash -l {0}
+        run: |
+          conda install -c conda-forge rdkit
+      - name: Install dev dependencies
+        shell: bash -l {0}
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[dev]
+      - name: Show pip list
+        run: pip list
```

Run tests so problems in any os/python version combi can be found.
```suggestion
      - name: Show pip list
        run: pip list
      - name: Run tests
        run: pytest
```